joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstCreator.scala

```diff
@@ -100,12 +104,14 @@ class AstCreator(
       case CatchClause                                           => astForCatchClause(nodeInfo)
       case CatchDeclaration                                      => astForCatchDeclaration(nodeInfo)
       case PropertyDeclaration                                   => astForPropertyDeclaration(nodeInfo)
+      case AnonymousObjectMemberDeclarator                       => astForAnonymousObjectMemberDeclarator(nodeInfo)
```

Comment:
Does this need to be registered here? It seems to always be called directly under `astForAnonymousObjectCreationExpression`